src/Symfony/Component/Security/Core/Authentication/Token/RememberMeToken.php

```diff
@@ -52,9 +50,14 @@ public function getFirewallName(): string
         return $this->firewallName;
     }
 
+    /**
+     * @deprecated since Symfony 7.2
+     */
     public function getSecret(): string
     {
-        return $this->secret;
+        trigger_deprecation('symfony/security-core', '7.2', 'The "%s()" method is deprecated.', __METHOD__);
+
+        return $this->secret ?? base64_encode(random_bytes(8));
```

Comment:
Should the generated value be stored so that calling the method multiple times always returns the same value?

Reply:
Works for me, updated.